test/terraform_aws_logs_alb_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestTerraformAwsLogsAlb(t *testing.T) {
+	t.Parallel()
+
+	expectedLogsBucket := fmt.Sprintf("terratest-aws-logs-alb-%s", strings.ToLower(random.UniqueId()))
+	vpcName := fmt.Sprintf("terratest-vpc-alb-%s", strings.ToLower(random.UniqueId()))
```

Comment:
to keep the names consistent for the various resources in a test can you use a single generated names?

Reply:
I standardized the test name for various resources.

Tests still pass:

```
--- PASS: TestTerraformAwsLogsCloudtrail (55.25s)
--- PASS: TestTerraformAwsLogsCombined (619.51s)
--- PASS: TestTerraformAwsLogs (52.84s)
--- PASS: TestTerraformAwsLogsS3 (65.59s)
--- PASS: TestTerraformAwsLogsElb (79.10s)
--- PASS: TestTerraformAwsLogsConfig (198.03s)
--- PASS: TestTerraformAwsLogsNlb (268.79s)
--- PASS: TestTerraformAwsLogsAlb (306.43s)
--- PASS: TestTerraformAwsLogsRedshift (611.13s)
ok  	github.com/trussworks/terraform-aws-logs/test	1286.211s
```